python/tk_multi_publish2/base_hooks/collector.py

```diff
@@ -85,9 +97,25 @@ def __init__(self, parent, hook, items, name, **kwargs):
             else:
                 value = values.pop()
 
-            super(CollectorPlugin.PropertyWidget, self).__init__(
+            super(CollectorPlugin.PropertyWidgetBaseClass, self).__init__(
                 parent, hook, name, value, value_type, **kwargs)
 
+        @property
+        def items(self):
+            return self._items
+
+    class PropertyWidget(PropertyWidgetBaseClass):
+        """
+        A widget class representing an item property.
+        """
+        # Signal for when a property value has changed
+        value_changed = QtCore.Signal()
```

Comment:
I would probably keep this on the base class and use it for both value and field updates rather than having two different ones, since they effectively do the same thing.